pkgs/applications/printing/pappl/default.nix

```diff
@@ -21,6 +22,13 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-FsmR0fFb9bU9G3oUyJU1eDLcoZ6OQ2//TINlPrW6lU0=";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/michaelrsweet/pappl/commit/7ec4ce4331b6637c54a37943269e05d15ff6dd47.patch";
```

Comment:
```suggestion
      name = "file-offset-bits-64-linux.patch";
      url = "https://github.com/michaelrsweet/pappl/commit/7ec4ce4331b6637c54a37943269e05d15ff6dd47.patch";
```

Reply:
Thanks for the suggestion, I should have done that in the first place. Ammended my commit with the change.